openreview/conference/builder.py

```diff
@@ -337,6 +339,8 @@ def open_meta_reviews(self, name, due_date = None, public = False):
 
     def set_program_chairs(self, emails):
         self.__create_group(self.get_program_chairs_id(), self.id, emails)
+        ## Give program chairs admin permissions
+        self.__create_group(self.id, '~Super_User1', [self.get_program_chairs_id()])
```

Comment:
This is saying that the signature of the Program Chairs group is the super user. Check the function parameters: 

```
def __create_group(group_id, group_owner_id, members)
```